deliverables/README.md

```diff
@@ -21,48 +21,110 @@ For projects on Code Academy, or other external exercises, please just paste a l
 
 ### Section 1: The Front End
 #### Required - Answer these questions in the deliverables file, and complete the following. 
+
 What is the role of HTML in a web page?  
+HTML acts as the skelton of the web page.
+
 What is the role of CSS in a web page?  
-What is the role of Javascript in a web page?  
+CSS acts as the skin of the web page
 
+What is the role of Javascript in a web page?  
+Javascript adds behavior to your webpage allowing the user to interact with it. 
 ---
 
 ### Section 2: HTML and CSS 101
 
 #### meta
 Why do we separate HTML and CSS?  
-
+We seperate the two for effiency, accesbility, device compatibility.
 #### HTML
 What are classes and IDs (and how are they different)?  
-What are elements?  
+ID's are unique, classes are not unique.
+
+What are elements? 
+Elements are everything inbetween the tags.
+
 What are tags?  
+Are keywords surronded by ankle braclets. They come in pairs, a start and end one.
```

Comment:
They also help us define the structure of a page. For example, '<p>' indicated that the text following it will be a paragraph.